### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by PavlovaOlena**
[DEMO LINK](https://PavlovaOlena.github.io/layout_search-bar-airbnb/)
[TEST REPORT LINK](https://PavlovaOlena.github.io/layout_search-bar-airbnb/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -11,6 +11,33 @@
...
+      </form>
+      <form 
+        class="form-search" 
+        action="/" 
+        method="get"
+      >
+        <input 
+          type="search" 
+          class="input-search input-search-small" 
+          name="text"
```
Comment (polosanya) [inline src/index.html:36]:
```suggestion
          name="smallSearch" 
```
This name would be more correct

File: src/style.css [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 800;
+}
+
+.input-search {
+  font-family: "Avenir", sans-serif;
+  font-weight: 300;
+  display: block;
+  background: url(images/Search.svg) no-repeat left;
```
Comment (polosanya) [inline src/style.css:23]: Pay attention to CSS properties order
https://9elements.com/css-rule-order/

**Review by polosanya — CHANGES_REQUESTED** [review]
image
image

File: src/index.html [hunk 3]
```diff
@@ -11,6 +11,33 @@
...
-    <h1>Search bar airbnb</h1>
+    <div class="container">
+      <form 
+        class="form-search" 
+        action="/" 
+        method="get"
+      >
+        <input 
+          type="search" 
+          class="input-search input-search-big"
```
Comment (witflash) [inline src/index.html:22]: If you use modifier it should be `input-search--big` in BEM methodology.

File: src/style.css [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+  border: 1px solid #e1e7ed;
+  border-radius: 3px;
+  background: url(images/Search.svg) no-repeat left, linear-gradient(#fff 0%, #f6f6f7 100%);
+  background-position: 20px 20px, 0 0;
+  font-weight: bold;
+  outline: none;
+}
+
+.input-search-small:focus {
+  background: url(images/Search.svg) no-repeat left, linear-gradient(#fff 0%, #f6f6f7 100%);
```
Comment (witflash) [inline src/style.css:63]: There is no need to use `background` rule here because you have the same for `input-search:focus` already.

**Review by witflash — APPROVED** [review]
Good job 👍

**Pull request closed without merging**